frontend/mgramseva/android/app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +17,8 @@
     <application
         android:allowBackup="false"
         android:fullBackupContent="false"
-        android:label="mgramseva"
+        android:enableOnBackInvokedCallback="true"
+        android:label="mGramSeva"
```

Comment:
pls make sure that this won't affect the deployment of app in playStore

Reply:
No, It will not hamper. I have checked.